### PR TITLE
Login ui change

### PR DESCRIPTION
**Pull request by maulikdhameliya**

### CODE REVIEW

File: src/signed-out/LoginScreen.tsx [hunk 1]
```diff
@@ -1,134 +1,159 @@
...
+      <RoundTextInput
+        imageNeeded
+        placeholder="Email"
+        keyboardType="email-address"
+        source={require('../../assets/images/email_logo.png')}
+      />
+      <RoundTextInput
+        imageNeeded
+        placeholder="Password"
+        keyboardType="email-address"
```
[the remark this answers is not in the document]
Comment (maulikdhameliya) [reply src/signed-out/LoginScreen.tsx]: I will do

**Comment by maulikdhameliya** [comment]
@bhaskerreddyb please review code
**_Known issues_**
- Country picker only opening on Flag icon, will fix it
- Logout not implemented yet

**Comment by bhaskerreddyb** [comment]
The login screen looks good and now I am able to log in.
Register screen also good as with phone number, make sure to change css to match with login screen.
Also the same as forgot password screen too.

**Comment by maulikdhameliya** [comment]
@bhaskerreddyb 
```
The login screen looks good and now I am able to log in.
Register screen also good as with phone number, make sure to change css to match with login screen.
Also the same as forgot password screen too.
```

=> fixed above

**Comment by bhaskerreddyb** [comment]
@maulikdhameliya 
You might have received an email from Vasanth regrading Ziplin. He sent to this email.
1) Based on that change icons
2) Change background color based on it (Instead of on small dots)
3) make sure to align fonts and sizes with it
4) Make sure to add icons to input boxes in signup screen and forgot password screen.

I am pretty much good to merge these once you add these.

**Comment by maulikdhameliya** [comment]
> You might have received an email from Vasanth regrading Ziplin. He sent to this email.

> Based on that change icons
> Change background color based on it (Instead of on small dots)
> make sure to align fonts and sizes with it
> Make sure to add icons to input boxes in signup screen and forgot password screen.
> I am pretty much good to merge these once you add these.

Fixed above and pushed please check @bhaskerreddyb

**Pull request merged**